test/unit/util_test.js

```diff
@@ -520,7 +520,55 @@ describe('Util', function ()
           assert.strictEqual(url, testCase.result);
         })
       }
-  })
+  });
+
+  it("Util.isLoginRequest Test", function () {
```

Comment:
The test looks right, could you refactor it so `testCases` are ran as individual tests instead? You can use the [test above](https://github.com/snowflakedb/snowflake-connector-nodejs/pull/671/files#diff-f7fc5e0118097ba33301dd904921ab1c12e77854a9493c441aaf79aadb00be69R518) as a reference

Reply:
Refactored!